src/ProcessMaker/Nayra/Bpmn/EndEventTrait.php

```diff
@@ -53,6 +53,9 @@ function (TransitionInterface $transition, CollectionInterface $consumeTokens) {
                 $this->notifyEvent(EventInterface::EVENT_EVENT_TRIGGERED, $this, $transition, $consumeTokens);
             }
         );
+
+        $this->triggerPlace = new State($this, GatewayInterface::TOKEN_STATE_INCOMMING);
```

Comment:
Do not use a TOKEN_STATE from other bpmn element.
Fix the spelling for "incomming" (should be incoming)